colibre/scripts/last_SNII_density_distribution.py

```diff
@@ -115,12 +115,28 @@ def critical_density_DVS2012(
     stars_SNII_densities = (snapshot.stars.densities_at_last_supernova_event / mh).to(
         SNII_density_bins.units
     )
-    stars_SNII_redshifts = 1 / snapshot.stars.last_sniifeedback_scale_factors.value - 1
+
+    # swift-colibre master branch as of Feb 26 2021
+    try:
+        stars_SNII_redshifts = (
+            1 / snapshot.stars.last_sniithermal_feedback_scale_factors.value - 1
+        )
+    # swift-colibre master prior to Feb 26 2021
+    except AttributeError:
```

Comment:
This is to have compatibility with the previous version of the swift master; I will remove it later.